ESP_MQTT_Digital_LEDs/ESP_MQTT_Digital_LEDs.ino

```diff
@@ -23,6 +23,8 @@
 #include <ArduinoJson.h>
 #include <ESP8266WiFi.h>
 #include <PubSubClient.h>
+// Uncomment this if you are experiencing flickering
+//#define FASTLED_ALLOW_INTERRUPTS 0
```

Comment:
This has been documented to trigger watchdog resets. Better way is to use this (https://github.com/coryking/FastLED) version of the fastled lib with esp8266 support (no serial support, maby add telnet debugging?)